src/test/crimson/test_config.cc

```diff
@@ -1,27 +1,81 @@
 #include <chrono>
+#include <string>
 #include <numeric>
 #include <seastar/core/app-template.hh>
 #include <seastar/core/sharded.hh>
+#include "common/config_obs.h"
 #include "crimson/common/config_proxy.h"
 
 using Config = ceph::common::ConfigProxy;
+const std::string test_uint_option = "osd_max_pgls";
+
+class ConfigObs : public ceph::internal::md_config_obs_impl<Config> {
+  uint64_t last_change;
+  uint64_t num_changes;
+  Config &local_conf;
+  const char **conf_keys;
```

Comment:
put
```c++
std::unique_ptr<const char*[]> conf_keys;
```
and initialize it in-class, like
```c++
std::unique_ptr<const char*[]> conf_keys = std::make_unique<const char*[]>(2);
```

Reply:
Was referencing code like https://github.com/ceph/ceph/blob/master/src/common/WorkQueue.h#L88... And yeah, this is better.